src/screens/TopicPageScreen/index.js

```diff
@@ -460,6 +439,10 @@ const TopicPageScreen = (props) => {
           onScroll={handleScroll}
           scrollEventThrottle={16}
           showSearchBar={true}
+          onMomentumScrollEnd={(event) => {
+            onWillSendViewPostTime(event, feeds);
+            fetchNextFeeds(event);
+          }}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onMomentumScrollEnd` event handler has been added to the component. This is a good practice as it allows you to perform actions when the scrolling animation ends. However, make sure that the functions `onWillSendViewPostTime` and `fetchNextFeeds` are properly defined and handle potential errors to avoid unexpected behavior.

```javascript
onMomentumScrollEnd={(event) => {
  onWillSendViewPostTime(event, feeds);
  fetchNextFeeds(event);
}}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->